Knowledge-catalog-challenge-lab-best-practices.md

```
@@ -0,0 +1,247 @@
+# Knowledge Catalog チャレンジラボ攻略ガイド
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)